config/config.go

```diff
@@ -919,6 +932,13 @@ func PrintPelicanVersion(out *os.File) {
 	fmt.Fprintln(out, "Built By:", GetBuiltBy())
 }
 
+func LogPelicanVersion() {
+	log.Infoln("Version:", GetVersion())
+	log.Infoln("Build Date:", GetBuiltDate())
+	log.Infoln("Build Commit:", GetBuiltCommit())
+	log.Infoln("Built By:", GetBuiltBy())
```

Comment:
I personally don't think the "Built By" line is very useful (it's always going to be goreleaser). Let's leave it in the old `PrintPelicanVersion()` function, but remove it from your new logging function.